src/dos/dos_locale.cpp

```diff
@@ -1549,6 +1651,11 @@ void DOS_Locale_AddMessages()
 	MSG_Add("DOSBOX_HELP_LIST_KEYBOARD_LAYOUTS_2",
 	        "The above codes can be used in the 'keyboard_layout' config setting.");
 
+	MSG_Add("DOSBOX_HELP_LIST_CODE_PAGES_1",
+	        "List of available code pages and relevant writing scripts");
```

Comment:
"relevant writing scripts" — what's that?

Reply:
I meant Latin, Greek, Armenian, etc. Removed this part.